support/Pipelines/Bbh/Ringdown.py

```diff
@@ -168,21 +168,28 @@ def start_ringdown(
     )
     match_time = evaluated_fot_dict["MatchTime"]
 
-    # This section checks for functions of time in the dictionary and alters
-    # some values in the evaluated functions of time dictionary. The ringdown
-    # only requires the outer boundary expansion map so here we set the inner
-    # expansion map to the identity. We also set the translation map to 0 since
-    # translation is not supported in the transition to ringdown yet.
-    if (
-        evaluated_fot_dict.get("Expansion") == None
-        or evaluated_fot_dict.get("ExpansionOuterBoundary") == None
-    ):
+    # This section checks for functions of time in the dictionary and changes
+    # them from tuples to lists and or lists of lists. It also alters some
+    # values in the evaluated functions of time dictionary. The ringdown only
+    # requires the outer boundary expansion map so here we set the inner
+    # expansion map to the identity (this should be changed to a settle to const
+    # at a later time). We also set the translation map to 0 since translation
+    # is not supported in the transition to ringdown yet.
+    if "ExpansionOuterBoundary" in evaluated_fot_dict:
+        evaluated_fot_dict["ExpansionOuterBoundary"] = [
+            x[0] for x in evaluated_fot_dict["ExpansionOuterBoundary"]
+        ]
+    else:
         raise ValueError(
-            f"The transition to ringdown script requires the Expansion and"
+            f"The transition to ringdown script requires the "
             f" ExpansionOuterBoundary functions of time to be valid at the"
             f" match time. "
         )
-    if evaluated_fot_dict.get("Rotation") == None:
+    if "Rotation" in evaluated_fot_dict:
+        evaluated_fot_dict["Rotation"] = [
+            [coef for coef in x] for x in evaluated_fot_dict["Rotation"]
+        ]
```

Comment:
Is this necessary? I forget what the python return type from `func_and_two_derivs` is.

Reply:
I wondered the same, but it is needed. It returns a list of DataVectors and the StrahlkorperCoefsInRingdownDistortedFrame call requires the FoTs to be a list of doubles (for expansion/expansion outer boundary) or list of list of doubles for all the other FoTs.

Reply:
You could covert everything to a list of lists inside `functions_of_time_from_volume`, then you don't have to worry about it here.